cmd/destroy.go

```diff
@@ -194,24 +194,28 @@ func destroyFn(_ *cobra.Command, _ []string) error {
 	}
 
 	// Remove any tool containers associated with destroyed labs
-	if !all && len(labs) > 0 {
-		// Get the lab name from the first lab
-		labName := labs[0].Config.Name
-		log.Debugf("Looking for tool containers with lab name: %s", labName)
-
-		// Initialize runtime
+	if len(labs) > 0 {
```

Comment:
@vista- I think this does not address the linked issue...

```
dclab dep -c -t srlinux.dev/clab-srl2
15:54:22 INFO Containerlab started version=0.0.0
15:54:22 INFO Parsing & checking topology file=topo-1646392938.clab.yml
15:54:22 INFO Destroying lab name=srl2
15:54:22 INFO Removed container name=srl1
15:54:22 INFO Removed container name=srl2
15:54:22 INFO Removing host entries path=/etc/hosts
15:54:22 INFO Removing SSH config path=/etc/ssh/ssh_config.d/clab-srl2.conf
15:54:22 INFO Removing directory path=/tmp/.clab/clab-srl2
          
   ERROR  
          
  The 'srl2' lab has already been deployed. Destroy the lab before deploying a lab with the same name.                

  Try --help for usage.
```

Reply:
I think this is fixed now!